src/app/landing/LandingPage.jsx

```diff
@@ -99,10 +109,15 @@ class LandingPage extends Component {
             </div>
 
             <picture>
+              <source
+                className="OnboardingImage"
+                media="(min-aspect-ratio: 2/3)"
+                srcSet="img/landing_page/step3.png"
+              />
               <img
                 className="OnboardingImage"
-                src="img/landing_page/step3.png"
-                alt=""
+                src="img/landing_page/step3_mobile.png"
+                alt="map showing food, clothing, and personal care markers"
```

Comment:
It's also supposed to show shelter IIRC, unless we never updated it.

Reply:
Will add!